TAO/tests/Bug_3542_Regression/DllOrb.h

```diff
@@ -25,7 +25,7 @@ class bug_3542_Export DllOrb: public ACE_Task_Base
   virtual int svc ();
 
 private:
-  auto_ptr < ACE_Thread_Barrier >      ma_barrier_;
+  std::unique_ptr < ACE_Thread_Barrier >    ma_barrier_;
```

Comment:
No need for spaces between `<> `and `ACE_Thread_Barrier` (only `<::` is problematic with some compilers, we always use `< ::` in that case), align `ma_barrier_` with the other members

Reply:
Would you like to have a .clang-format to enforce our code style?

Reply:
Feel free to add one in a different PR

Reply:
OK